pkgs/development/interpreters/racket/minimal.nix

```diff
@@ -2,9 +2,9 @@
 }:
 
 racket.overrideAttrs (oldAttrs: rec {
-  name = "racket-minimal-${oldAttrs.version}";
+  pname = "racket-minimal";
+  version = oldAttrs.version;
   src = oldAttrs.src.override {
-    inherit name;
```

Comment:
Not sure if we want to inherit pname + version here.

Reply:
I am not quite sure what you mean here

Reply:
```suggestion
    inherit pname version;
```

Reply:
sorry but i still do not get it. The snipped does not work since the arguments are not expected.

Reply:
Ahhh, yeah that makes sense.

I am not sure if we want to do then the following or if it does not matter.

```suggestion
    name = "${pname}-${version}";
```